.github/workflows/update.yml

```diff
@@ -48,9 +49,19 @@ jobs:
           name: nargo-cache
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
-      - name: Build environment and Compile
+      - name: Build with Nix
         run: |
-          nix-shell noir_wasm.nix --pure --run "./build.sh"
+          nix build -L github:noir-lang/noir/jb/noir-wasm-workflow#wasm --extra-experimental-features nix-command --extra-experimental-features flakes

```

Comment:
Are `--extra-experimental-features nix-command --extra-experimental-features flakes` needed? They should be removed...